src/app/js/admin/parsing/ParsingEditCell.js

```diff
@@ -28,71 +30,207 @@ const style = {
     },
     containedButton: {
         marginTop: 15,
-        float: 'right',
     },
     icon: {
         marginRight: 10,
     },
+    menuPaper: {
+        backgroundColor: theme.green.primary,
+        '& .MuiMenuItem-root': {
+            color: 'white',
+        },
+    },
 };
 const useStyles = makeStyles(style);
 
-const returnParsedValue = value => {
+export const returnParsedValue = value => {
     try {
         return JSON.parse(value);
     } catch (e) {
         return value;
     }
 };
 
-const isPrimitive = value => {
+export const isPrimitive = value => {
     return value !== Object(value);
 };
 
+export const tryParseJSONObjectOrArray = jsonString => {
+    try {
+        if (typeof jsonString === 'object') {
```

Comment:
is jsonString can be null ? because typeof something null will be an object

Reply:
If jsonString is null, it will return himself.